src/library.js

```diff
@@ -3007,6 +3007,13 @@ LibraryManager.library = {
     var info = _mmap.mappings[start];
     if (!info) return 0;
     if (num == info.num) {
+      // At the Linux man page, it says:
+      // "The file may not actually be updated until msync(2) or munmap(2) are called."
+      // I guess that means we need to call msync when doing munmap
+      _msync(start, num); // todo: which flags?
```

Comment:
Is there any condition/combination flags that could be used to avoid calling `msync` unconditionally here? (e.g. read-only mmap would not need this?)